common/netlog.py

```diff
@@ -116,6 +117,20 @@ def resolve_flags(self, apiname, argdict, flags):
 
             flags[argument] = "|".join(flags[argument])
 
+    @staticmethod
+    def _normalize_arguments(raw_arguments):
+        """
+        Decodes and formats hexstreams in arguments into human-readable ASCII if possible
+        :param raw_arguments: Dict
+        :return: Dict
+        """
+        arguments = {}
+
+        for key, value in raw_arguments.iteritems():
```

Comment:
consider one-liner:
arguments = { k: handle_hex_stream(v) for k,v in raw_arguments.items() }